pkg/manager/manager.go

```diff
@@ -64,9 +58,8 @@ type DeleteClusterOption struct {
 }
 
 type CreateOperatorOptions struct {
-	Namespace       string
```

Comment:
It's not the correct implementation. If the user specifies the version, use the specific version; if not, use the latest version. DON'T delete the version field.